xep-0313.xml

```diff
@@ -169,6 +177,17 @@
 	<section2 topic='Querying Entities' anchor='entities'>
 		<p>While this document talks about 'clients' and 'servers', as these are the common cases, the querying entity (referred to as a 'client') need not be an XMPP client as defined by RFC6120, but could potentially be any type of entity, and the queried entity (referred to as a 'server') need not be an XMPP server as defined by RFC6120, although access controls might prohibit any given entity from being able to access an archive.</p>
 	</section2>
+	<section2 topic='Communicating the archive ID' anchor='archives_id'>
+		<p>When an incoming message is archived, the server MAY add an &lt;stanza-id/&gt; element as defined in &xep0359; to the message, which informs the client of where and under what ID the message is stored. When doing this the server MUST follow the business rules defined in XEP-0359. The 'by'-attribute MUST be set to the address of the archive. For regular users that’s the bare JID of the account and for MUC that’s the bare JID of the room.</p>
+		<p>Servers MUST NOT include the &lt;stanza-id/&gt; element in messages addressed to JIDs that do not have permissions to access the archive, such as a users’s outgoing messages to their contacts. However servers SHOULD include the element as a child of the forwarded message when using &xep0280;</p>
```

Comment:
@Kev What do you think about s/MAY/SHOULD/ and not bumping namespace? I've been working on a new revision with a collection of changes, and have persuaded myself for now that keeping the current namespace is best. I don't think this change will change my mind on that, unless you have a convincing argument (which I'd be willing to hear).